_knitr_hooks.qmd

```diff
@@ -0,0 +1,32 @@
+```{r}
+#| echo: false
+
+
+  # Remove personal directories
+redact <- function(x) {
+  stringr::str_replace_all(
+    x,
+    "/Users/jkeane.*/(PUMS_smol|PUMS cleaned)/",
```

Comment:
```suggestion
    "/Users/(jkeane|npr).*/(PUMS_smol|PUMS cleaned)/",
```

Reply:
@thisisnic are you usually at `/home/???`? I can add that in too

Reply:
Yup, `/home/nic` for me!